share/picongpu/dockerfiles/README.rst

```diff
@@ -25,7 +25,7 @@ This exposes the ISAAC port to connect via the webclient to.
 .. code:: bash
 
     docker pull ax3l/picongpu
-    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.5.0 lwfa_live
+    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.6.0-dev lwfa_live
```

Comment:
I reserved an official `picongpu/...` org on dockerhub a while back as well as a `computationalradiationphysics/` prefix.

Please tell me your dockerhub user names, so I can add you as co-owners and you can then update these lines and push new docker images to it.

Reply:
my dockerhub user name is `psychocoderhpc`

Reply:
@ax3l do you reserved it for https://singularity-hub.org/ too?

Reply:
I opened #3544 to keep track that this will be fixed.

Reply:
> my dockerhub user name is psychocoderhpc

thanks, added as owner!

> singularity hub

Nope, none there. At some point SH did only allow building on their infrastructure but not pushing pre-build images - and resources to built on their platform were too small to get the whole stack built there. Maybe that changed in between, not sure.